charts/nri-prometheus/static/guidedinstalldefaults.yaml

```
@@ -0,0 +1,134 @@
+# many metrics are being scraped.
+# Default: 4
+# worker_threads: 4
+
+# Maximum number of metrics to keep in memory until a report is triggered.
+# Changing this value is not recommended unless instructed by the New Relic support team.
+# max_stored_metrics: 10000
+
+# Minimum amount of time to wait between reports. Cannot be lowered than the default, 200ms.
+# Changing this value is not recommended unless instructed by the New Relic support team.
+# min_emitter_harvest_period: 200ms
+
+# targets:
+#   - description: Secure etcd example
+#     urls: ["https://192.168.3.1:2379", "https://192.168.3.2:2379", "https://192.168.3.3:2379"]
+#     tls_config:
+#       ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+#       cert_file_path: "/etc/etcd/etcd-client.crt"
+#       key_file_path: "/etc/etcd/etcd-client.key"
+
+# Proxy to be used by the emitters when submitting metrics. It should be
+# in the format [scheme]://[domain]:[port].
+# The emitter is the component in charge of sending the scraped metrics.
+# This proxy won't be used when scraping metrics from the targets.
+# By default it's empty, meaning that no proxy will be used.
+# emitter_proxy: "http://localhost:8888"
+
+# Certificate to add to the root CA that the emitter will use when
+# verifying server certificates.
+# If left empty, TLS uses the host's root CA set.
+# emitter_ca_file: "/path/to/cert/server.pem"
+
+# Set to true in order to stop autodiscovery in the k8s cluster. It can be useful when running the Pod with a service account
+# having limited privileges.
+# Default: false
+# disable_autodiscovery: false
+
+# Whether the emitter should skip TLS verification when submitting data.
+# Default: false
+# emitter_insecure_skip_verify: false
+
+# Histogram support is based on New Relic's guidelines for higher
+# level metrics abstractions https://github.com/newrelic/newrelic-exporter-specs/blob/master/Guidelines.md.
+# To better support visualization of this data, percentiles are calculated
+# based on the histogram metrics and sent to New Relic.
+# By default, the following percentiles are calculated: 50, 95 and 99.
+#
+# percentiles:
+#   - 50
+#   - 95
+#   - 99
+#
+# integration definition files required to map metrics to entities
+# definition_files_path: /etc/newrelic-infra/definition-files
+
+transformations:
+  - description: "New Relic Guided Install Defaults"
+    rename_attributes:
+      - metric_prefix: ""
+        attributes:
+          container_name: "containerName"
+          pod_name: "podName"
+          namespace: "namespaceName"
+          node: "nodeName"
+          container: "containerName"
+          pod: "podName"
+          deployment: "deploymentName"
+    ignore_metrics:
+      # Ignore the following metrics.
+      # These metrics are already collected by the New Relic Kubernetes Integration.
+      - prefixes:
+        - kube_
+        - container_
+        - machine_
+        - cadvisor_
```

Missing newline before EOF

```suggestion
        - cadvisor_

```